apps/reference/docs/guides/auth/server-side-rendering.mdx

```diff
@@ -0,0 +1,189 @@
+`my-refresh-token` cookies. (Do change the names of the cookies and additional
+parameters as they suit you.)
+
+In your server-side rendering code you can now access user and session
+information in the following way:
+
+
+```typescript
+const refreshToken = req.cookies['my-refresh-token'];
+
+if (refreshToken) {
+  supabase.auth.setSession(refreshToken);
+} else {
+  // make sure you handle this case!
+  throw new Error("User is not authenticated.")
+}
+
+await supabase.auth.getUser(); // will return user information
+```
+
+We recommend using the `setSession(refreshToken)` method instead of
+`getUser(accessToken)`, as refresh tokens are long lived credentials that will
+give your server user information even if the user has not user your app in a
+long while.
+
+:::tip
+Even though refresh tokens are long lived, they are no guarantee that a user
+has an active session. They may have logged out and your application failed to
+remove the `my-refresh-token` cookie, or some other unpredicted failure
+occurred that left a stale refresh token in the browser. **A good rule-of-thumb
+is to handle unauthorized errors by deferring to render the page in the browser
+instead of in the server.** Some user information is contained in the access
+token though, so in certain cases you may be able to use this potentially stale
+information to render a page.
+:::
+
+## Frequently Asked Questions
+
+### How do I make the cookies `HttpOnly`?
+
+This is not necessary. Both the access token and refresh token are designed to
+be passed around to different components in your application. The browser based
+side of your application needs access to the refresh token to properly maintain
+a browser session anyway.
+
+### My server is getting invalid refresh token errors. What's going on?
+
+It is likely that the refresh token sent from the browser to your server is
+stale. Make sure the `onAuthStateChange` listener callback is free of bugs and
+is registered relatively early in your application's lifetime.
+
+If possible, when you receive this error on the server side try to defer
+rendering to the browser where the client library can access an up-to-date
+refresh token and present the user with a better experience.
+
+### Should I set a shorter `Max-Age` parameter on the cookies?
+
+The `Max-Age` or `Expires` cookie parameters only control whether the browser
+will send the value to the server. Since a refresh token represents the
+long-lived authentication session of the user on that browser, setting a short
+`Max-Age` or `Expires` parameter on the cookies will only result in a degraded
+user experience.
+
+The only way to ensure that a user has logged out or their session has ended in
+some way is to check with a call to `getUser()`.
+
+### What should I use for the `SameSite` property?
+
+Make sure you understand the parameter correctly, as your users can be
+experiencing a degraded user experience.
+
+A good default is to use `lax`.
```

Comment:
Can we explain what this property does or link out to something? Maybe https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite?

Reply:
Yes.